src/calculator.py

```diff
@@ -0,0 +1,27 @@
+def sum(a: int, b: int) -> int:
+    return a + b
+
+
+def diff(a: int, b: int) -> int:
+    return a - b
+
+
+def mol(a: int, b: int) -> int:
+    return a * b
+
+
+def div(a: int, b: int) -> int:
+    return a // b
+
+
+def main():
+    a = int(input("Insert first number: "))
+    b = int(input("Insert second number: "))
+
+    print("SUM:", sum(a, b))
+    print("Difference:", diff(a, b))
+    print("Moltiplication:", mol(a, b))
+    print("Division:", div(a, b))
+
+
+main()
```

Comment:
Aggiungi un 
```python
if __name__ == "__main__":
    main()
```
come [qui](https://www.geeksforgeeks.org/what-does-the-if-__name__-__main__-do)